scout/server/links.py

```diff
@@ -101,6 +102,16 @@ def gnomad_str_gene(hgnc_symbol):
     return link.format(hgnc_symbol)
 
 
+def gnomad_sv_gene(hgnc_symbol):
```

Comment:
We don't need this one no? I confused variant and gene links

Reply:
Right, I dont think so!

Reply:
Removed